pytorch_lightning/trainer/properties.py

```diff
@@ -11,11 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import inspect
-import os
 from abc import ABC
 from argparse import ArgumentParser, Namespace
-from typing import List, Optional, Type, TypeVar, Union, cast
+import inspect
+import os
```

Comment:
there is something strange with your ordering... or do we have it in PL setting?
it shall `import ...` and then `from ...`

Reply:
done